plugins/python-build/bin/python-build

```diff
@@ -473,6 +473,11 @@ download_tarball() {
   fi
 }
 
+has_tar_xz_support() {
+  local tarxz="$(mktemp "XXXXXXXX.tar.xz")"
```

Comment:
It does not work on NetBSD/OpenBSD.  They assume "XXXXXXXX" is at the end of the template string.

NetBSD:
```
% mktemp XXXXXXXX.tar.xz
XXXXXXXX.tar.xz
```

OpenBSD:
```
% mktemp XXXXXXXX.tar.xz
mktemp: insufficient number of Xs in template `XXXXXXXX.tar.xz'
```

But we don't have to create a temporary file to check if `tar` command supports `-J` option.  With `-f -` option, `tar` writes an archive to stdout.  I tried ``tar Jcf - /dev/null 1>/dev/null 2>&1`` and it works on FreeBSD/NetBSD/OpenBSD/macOS/Ubuntu ($?=0 on FreeBSD/macOS/Ubuntu and $?=1 on NetBSD/OpenBSD).